tests/test_scrapy.py

```diff
@@ -60,14 +60,14 @@ def test_request_response_converters():
 
     frontier_response = rsc.to_frontier(response)
     assert frontier_response.body == RESPONSE_BODY
-    assert frontier_response.meta[b'scrapy_meta'][b'test_param'] == b'test_value'
+    assert frontier_response.meta[b'scrapy_meta']['test_param'] == b'test_value'
```

Comment:
@sibiryakov what do you think of this particular assertion? should we have the entirety of `meta` as bytes when sending over to Frontera?